apps/site/assets/ts/helpers/prediction-helpers.tsx

```diff
@@ -27,7 +27,11 @@ export const timeForCommuterRail = (
     return delayForCommuterRail(data, className);
   }
 
-  const time = prediction ? prediction.time : scheduledTime;
+  let time = prediction ? prediction.time : scheduledTime;
+  if (scheduledTime && prediction) {
+    // "On time"
```

Comment:
I feel like there should be a more granular way to conclude whether the train is "on time" or not? 🤔 Also this would show the `scheduledTime` even if there's a delay of 4 or fewer minutes, which doesn't seem quite desirable. 

I think what might make sense is to show the prediction time only if it happens to be later than the scheduled time.